### ISSUE TITLE
Not working in NodeJS <12.16.0

### ISSUE THREAD

**Issue by chenaski** [ai/check-dts#21]
I faced this error in CI pipeline on Node 12.5.0:

```bash
/app/node_modules/check-dts/bin.js:3
import { red } from 'colorette'
^^^^^^

SyntaxError: Cannot use import statement outside a module
```

Reproduce:

```bash
for node_version in {2..22}; do docker run --rm -it node:12.$node_version-alpine sh -c "node -v && npm -v && npx --node-arg=--experimental-modules check-dts"; done
```

In [package.json](https://github.com/ai/check-dts/blob/be34387333b3b25ca1859273d873c8682eacdba1/package.json#L14) specifies `"node": ">=12.2.0"`, but NodeJS version in [workflow](https://github.com/ai/check-dts/blob/be34387333b3b25ca1859273d873c8682eacdba1/.github/workflows/test.yml#L28) resolves to the [latest minor](https://github.com/ai/check-dts/runs/3031490007?check_suite_focus=true#step:3:9).

We can make a cjs version to support a wider range of NodeJS, or we can just increment the NodeJS version in package.json and specify the exact version in the workflow so the tests are "fair".

**Comment by ai** [comment]
Let's increase version limit. Do you want to send PR?

**Comment by chenaski** [comment]
Yeap 🙂

**Comment by ai** [comment]
Released in 0.5.3
